src/main/java/uk/gov/hmcts/reform/roleassignment/controller/endpoints/DeleteAssignmentController.java

```diff
@@ -120,9 +128,58 @@ public DeleteAssignmentController(@Autowired DeleteRoleAssignmentOrchestrator de
     public ResponseEntity<Void> deleteRoleAssignmentById(@RequestHeader(
         value = "x-correlation-id",
         required = false)
-                                                               String correlationId,
-                                                           @ApiParam(value = "assignmentId", required = true)
-                                                           @PathVariable String assignmentId)  {
+                                                             String correlationId,
+                                                         @ApiParam(value = "assignmentId", required = true)
+                                                         @PathVariable String assignmentId) {
         return deleteRoleAssignmentOrchestrator.deleteRoleAssignmentByAssignmentId(assignmentId);
     }
+
+    @PostMapping(
+        path = "/am/role-assignments/query/delete",
+        consumes = V1.MediaType.DELETE_ASSIGNMENTS_BY_QUERY,
```

Comment:
We don't need to restrict the media type for only this one selected. The endpoint should only produces this new media type.

Reply:
this content-type is specified in api specification below.
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1504216881

Reply:
Need clarity on this.